plugins/xtrace/wiring.go

```diff
@@ -3,7 +3,7 @@
 //  1. A modifier plugin to wrap the service with an XTrace wrapper to generate XTrace compatible traces/logs by correctly propagating baggage across service boundaries. Automatically generates and include an xtrace-server instance in a Blueprint application.
 //  2. A plugin to define an xtrace-based logger for a process. Log events are added as reports to the currently active xtrace task, if one exists. If no such task exists, then no events are logged.
```

Comment:
I missed this in the x-trace PR, so update it here:

The package comment should be a single sentence overviewing the package.  In the go.dev APIs (and elsewhere), the first sentence of the package comment is typically used as a brief description. So for this package, you would simply see "Package xtrace provides three plugins:" which is not useful.  Change this to just summarize the package:  

> Package xtrace provides a Blueprint plugin for instrumenting services and collecting X-Trace traces.

I wouldn't refer to each piece of functionality as a separate plugin, tbh. This is one plugin that does several things.